src/check_impl.h

```diff
@@ -65,6 +65,8 @@ struct TCase
     List *unch_tflst;
     List *ch_sflst;
     List *ch_tflst;
+    unsigned int num_tags;
+    char **tags;
```

Comment:
Check has a list structure available in [check_list.c](https://github.com/libcheck/check/blob/master/src/check_list.c). Consider using that instead of an char**.